crates/runtime/src/dataconnector/github/mod.rs

```diff
@@ -703,6 +694,9 @@ pub(crate) fn filter_pushdown(expr: &Expr) -> FilterPushdownResult {
             let parameter = match column_name {
                 "title" => format!("{value} in:title"),
                 "body" => format!("{value} in:body"),
+                "state" => format!("is:{value}"), // is:merged, is:closed, is:open provides more granular results than state:closed
```

Comment:
Should these be uppercase? GitHub states are like `OPEN` `CLOSED` etc.

Reply:
no, search uses lowercase values for modifiers like these. That's something I've experienced working on this, the state value is uppercase but the search value is lowercase.